app.js

```diff
@@ -0,0 +1,100 @@
+const express = require("express");
+const cors = require("cors");
+const bodyParser = require("body-parser")
+const School = require( "./School.js")
+
+const port = 3000;
+
+const app = express();
+
+let mySchool = new School()
+
+app.use(cors());
+app.use(bodyParser.urlencoded({extended: false}));
+app.use(bodyParser.json())
+app.listen(port, () => {
+    console.log("Listening to port ", port)
+})
+
+
+const addNewClass = (req,res,next) =>{
+    let newClassName = req.body["name"]
+    let newClassTeacher = req.body["teacher"]
+    
+    if(mySchool.classes[newClassName]){
+        res.json({status:"failure", message: "Class already exist"})
+    } else {
+        mySchool.addClass(newClassName,newClassTeacher)
+        console.log(mySchool)
+        res.json({
+            class:mySchool.classes[newClassName],
+            status:"sucess",
+            message: "Created a new class",
+            timestamp: Date(Date.now()).toString()
+        })
+    }
+}
+
+app.post("/school/classes", addNewClass)
```

Comment:
if `addNewClass` is just your callback, then you wouldnt need a `next()` argument